roles/edpm_neutron_dhcp/tasks/install.yml

```diff
@@ -19,15 +19,19 @@
     path: "{{ item.path }}"
     setype: "container_file_t"
     state: directory
-    owner: "{{ item.owner | default(lookup('env', 'USER')) }}"
```

Comment:
Please revert this. The `ansible_user` var isn't safe, we need something that works all the time. You can change the user it's executed under instead.

Reply:
Done